.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Vault UI Tests
+
+on:
+  workflow_call:
+    inputs:
+      test_filter:
+        type: string
+        description: "A filter to limit the ui tests to. Will be appended to the ember test command as '-f=<filter>'"
+        required: false
+      storage_backend:
+        type: string
+        description: "The storage backend to use, either 'raft' or 'consul'"
+        default: raft
```

Comment:
Just cause I know you'll like this. If we change this to `workflow_dispatch` we can actually give people `choice` types!

```yaml
    inputs:
      storage_backend:
        description: The Vault storage backend
        required: true
        default: raft 
        type: choice
        options:
        - raft
        - consul
```

Reply:
awesome, I like it.